### PR TITLE
added severities to logger (close #36)

### LINKED ISSUE #36: Add exception severities to logger

### PR DESCRIPTION
**Pull request by aurelenc**

### CODE REVIEW

File: Src/Core/Logger.cpp [hunk 1]
```diff
@@ -13,18 +13,56 @@
 #include <sstream>
 #include <string>
 
-ArcadeException Logger::log(ArcadeException ex)
+static std::string getSeverityColor(Logger::Severity severity)
+{
+    switch (severity) {
```
Comment (Cavonstavant) [inline Src/Core/Logger.cpp:18]: do an array

File: Src/Core/Logger.cpp [hunk 2]
```diff
@@ -13,18 +13,56 @@
...
+        case Logger::SEVERITY_INFO:
+            return LOGGER_COLOR_CYAN;
+        default:
+            return LOGGER_COLOR_RESET;
+    }
+}
+
+static std::string getSeverityString(Logger::Severity severity)
+{
+    switch (severity) {
```
Comment (Cavonstavant) [inline Src/Core/Logger.cpp:36]: Same

File: Src/Core/Logger.hpp [hunk 3]
```diff
@@ -13,18 +13,39 @@ class ArcadeException;
 /// \brief Logger class to log exceptions
 class Logger {
     public:
+        enum Severity {
```
Comment (Cavonstavant) [inline Src/Core/Logger.hpp:16]: Explicit enum values not needed
Add enum size type

**Review by Cavonstavant — CHANGES_REQUESTED** [review]

**Review by pariciadra — APPROVED** [review]

File: Src/Core/Logger.cpp (hunk 1 above)
Comment (aurelenc) [reply Src/Core/Logger.cpp:18]: no because it uses an enum

File: Src/Core/Logger.cpp (hunk 2 above)
Comment (aurelenc) [reply Src/Core/Logger.cpp:36]: Same

**Review by Cavonstavant — APPROVED** [review]

**Pull request merged**